db/migrate/20190814131051_create_credentials.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCredentials < ActiveRecord::Migration[5.2]
+  def change
+    create_table :credentials do |t|
+      t.references :user, foreign_key: true
+      t.string :external_id, null: false
+      t.text :public_key, null: false
```

Comment:
- There [SHOULD](https://www.w3.org/TR/webauthn-2/#sctn-sign-counter) be a `sign_count`.
- A name to help keep keys apart (assuming multiple keys will be allowed in the future)
- Also suggested on the W3C mailing list of GitHub repo as a best practice: a "last logged in at" timestamp to help disambiguate keys and potentially spot suspicious activity